src/main/scala/edg_ide/ui/tools/DefaultTool.scala

```diff
@@ -155,6 +155,11 @@ class DesignBlockPopupMenu(path: DesignPath, interface: ToolInterface)
       config.options.objectives = config.options.objectives ++ Seq(DseObjectiveFootprintCount(path))
       BlockVisualizerService(project).onDseConfigChanged(config)
     }, "Add objective contained footprint count"))
+    add(ContextMenuUtils.MenuItem(() => {
+      val config = BlockVisualizerService(project).getOrCreateDseRunConfiguration(rootClass)
+      config.options.objectives = config.options.objectives ++ Seq(DseObjectiveFootprintPrice(path))
+      BlockVisualizerService(project).onDseConfigChanged(config)
+    }, "Add objective contained footprint price"))
```

Comment:
```suggestion
    }, "Add objective price"))
```
This hasn't been merged yet but in a work-in-progress PR I've simplified the menu item text - this change would make this stylistically consistent with the new style.